packages/bindgen/src/templates/node-wrapper.ts

```diff
@@ -166,27 +186,36 @@ export function generate({ spec: rawSpec, file }: TemplateContext): void {
         ...params,
         asyncSig ? "_cb" : [],
       ].flat();
-      let call = `${native}(${args})`;
+      let call = `${native}([${args}])`;
       if (asyncSig) call = `_promisify(_cb => ${call})`;
-      body += `
+      const promisify_definition = `
```

Comment:
This should be called `method_definition` or just `definition`, since it isn't always promisified.